rust/noosphere-cli/README.md

```diff
@@ -5,3 +5,56 @@
 The Noosphere CLI is a reference client and pedagogical tool to demonstrate
 the principles of the Noosphere protocol and give interested users a
 no-code, low-complexity tool to synchronize content through the Noosphere.
+
+
+## Usage
+### User perspective
+```sh
+# Create an identity
+orb key create `whoami`
+
+# make a directory for your sphere.
+mkdir my-sphere
+cd my-sphere
+
+# Generate your personal sphere.
+orb sphere create --owner-key `whoami`
+
+# ..now make edits ..
+
+# see the status of files in your directory
+orb status
+
+# persist changes to the sphere
+orb save
+
+# join a gateway, after you've set this identity as the counterpart
```

Comment:
Wonder if we can just get the counterpart sphere from the gateway after setting, though makes things a bit magical. In lieu of that, maybe just have a line for that like `orb config set counterpart <counterpart_did>`

Reply:
Yah, this is room for refinement. Today it doesn't work that way, so the docs are accurate.